string result
float32 score
float32 x_coordinate
float32 y_coordinate
